content/fact/chrooted-webserver.md

```diff
@@ -9,7 +9,21 @@ webserver installation running.
 This is also true for HTTP daemons from packages (`apache2`, `nginx`).
 
 ```
-$ cat /etc/passwd
+$ userinfo www
+login   www
+passwd  *
+uid     67
+groups  www
+change  NEVER
+class
+gecos   HTTP Server
+dir     /var/www
+shell   /sbin/nologin
+expire  NEVER
+```
+
+```
+$ grep "www" /etc/passwd
 www:*:67:67:HTTP Server:/var/www:/sbin/nologin
```

Comment:
I find this too long and complicated to demonstrate what its for. 

Maybe only the `grep` part?

Reply:
yes, you can only use `grep`....
but, `userinfo` is more adequate. 

It's up to you! ;)